### ISSUE TITLE
datalad describe manages to add .git/ tree into git

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#252]
If I surround invocation of "datalad describe" with "git status" in make_pymvpa_collection.sh script (from #251) I get:
```
On branch master
nothing to commit, working directory clean
I: describing collection
On branch master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   ./.git/index
	modified:   ./.git/logs/HEAD
	modified:   ./.git/logs/refs/heads/master
	modified:   ./.git/refs/heads/master
	modified:   datalad.ttl

no changes added to commit (use "git add" and/or "git commit -a")
```
previously reported also in #246 ;-)

**Comment by bpoldrack** [comment]
I think it was fixed already, but not sure atm. I'll look into it again.

**Comment by hanke** [comment]
Not fixed in master at least.

We should try hard to get everything into master to have a common base for manual testing.

**Comment by bpoldrack** [comment]
Fixed in PR #248
